edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/lrec/PosWordConjunctionSizeTwoWindowSizeTwo.java

```diff
@@ -12,29 +12,22 @@
 
 import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
 import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
-import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Queries;
-import edu.illinois.cs.cogcomp.core.datastructures.textannotation.View;
 import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.View;
 import edu.illinois.cs.cogcomp.edison.features.DiscreteFeature;
 import edu.illinois.cs.cogcomp.edison.features.Feature;
 import edu.illinois.cs.cogcomp.edison.features.FeatureExtractor;
-import edu.illinois.cs.cogcomp.edison.features.RealFeature;
-import edu.illinois.cs.cogcomp.edison.features.helpers.SpanLabelsHelper;
 import edu.illinois.cs.cogcomp.edison.utilities.EdisonException;
 
-import java.util.*;
-
-import java.io.File;
-import java.io.FileWriter;
-import java.io.BufferedWriter;
-import java.io.IOException;
+import java.util.LinkedHashSet;
+import java.util.Set;
 
 /**
+ * Extracts the POS Tags as well as the form (text) of tokens 2 before and 2 after from
+ * the given token and generates a discrete feature from it.
  *
- * @author Paul Vijayakumar, Mazin Bokhari Extracts the POS Tags as well as the form (text) of
- *         tokens 2 before and 2 after from the given token and generates a discrete feature from
- *         it.
- *
+ * @keywords pos-tagger, words, tokens, window
+ * @author Paul Vijayakumar, Mazin Bokhari, Christos Christodoulopoulos
  */
 public class PosWordConjunctionSizeTwoWindowSizeTwo implements FeatureExtractor {
```

Comment:
Maybe make it a little more clear, cause it's confusing in this form; how about: `PosWordConjunctionTwoGramWindowTwo`